### PR TITLE
[TR] Clean Behat : Fix Permission decorator context and page

### PR DESCRIPTION
**Pull request by willy-ahva**
This PR clean the Permission behat cycle (Context / Decorator / Page) to fit exactly what we have to do and have a real good example.

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       |
| Added Behats                      |
| Changelog updated                 |
| Review and 2 GTM                  |
| Micro Demo to the PO (Story only) |
| Migration script                  |
| Tech Doc                          |

### CODE REVIEW

File: features/Pim/Behat/Context/Domain/System/PermissionsContext.php [hunk 1]
```diff
@@ -13,12 +13,27 @@ class PermissionsContext extends PimContext
      * @param string $permission
```
Comment (loicguillois) [inline features/Pim/Behat/Context/Domain/System/PermissionsContext.php]: $action

**Comment by loicguillois** [comment]
GTM

File: features/Pim/Behat/Context/Domain/System/PermissionsContext.php [hunk 2]
```diff
@@ -27,19 +42,39 @@ public function iSetRightsToACLResources($permission, $resources)
...
 
     /**
      * @param string $action (grant|remove)
      * @param string $group
      *
+     * @throws \InvalidArgumentException If $action is not a defined method
+     *
      * @When /^I (grant|revoke) rights to groups? (.*)$/
      */
     public function iSetRightsToACLGroups($action, $groups)
```
Comment (juliensnz) [inline features/Pim/Behat/Context/Domain/System/PermissionsContext.php]: we could regroup this method with the previous one. Not a big deal, just sad to have twice the same code :)

**Comment by juliensnz** [comment]
Really nice work, much cleaner like that :)

**Comment by ClementGautier** [comment]
GTM

**Pull request merged**